src/db/service/ExecutionService.ts

```diff
@@ -16,86 +15,80 @@ import {
   UnitTestOperations,
   TableGenericOperations,
   RecordMatches,
+  OutputFormat,
 } from '../entity/enum';
 import { store } from '../redux/store';
 import RowTestService from './RowTestService';
 import TableTestService from './TableTestService';
 
+const util = require('util');
+
 export default class ExecutionService {
   repository: Repository<ExecutionEntity>;
 
   constructor() {
     this.repository = AppDataSource.getRepository(ExecutionEntity);
   }
 
-  public async checkPassFail(
-    procedure: string,
-    parameters: string[],
-    test: ExecutionModelType
-  ) {
+  private getClient(database: string) {
+    return store.getState().connection.database.get(database);
+  }
+
+  public async checkPassFail(test: ExecutionModelType) {
```

Comment:
Since you changed this, you'll have to change the lines in ipc.ts, preload.ts and preload.d.ts in order for the checks to pass (take 1 arg instead of 3). Idk how to request changes on files outside a diff but should be pretty easy